src/util/address.rs

```diff
@@ -1273,4 +1314,68 @@ mod tests {
         assert_eq!(address.address_type(), Some(AddressType::P2tr));
         roundtrips(&address);
     }
+
+    mod test_is_related_to_secp256k1_key {
```

Comment:
What is the reason for adding this additional module? Could be removed and for each test below we could to `s/test_/is_related_to_key_/`

(In case you are interested; in this repo currently we use non-idiomatic prefix of tests `test_`, I'm personally trying to change that ... slowly)

Reply:
> What is the reason for adding this additional module?

No particular reason other than grouping similar test. I have no strong feelings here and can change to whatever is most consistent and preferred by everyone